.github/workflows/bump-channel-version/action.yml

```
@@ -21,8 +21,14 @@ runs:
   steps:
     - shell: bash
       run: |
-            CURRENT_CHANNEL_VERSION=${{ env.CURRENT_CHANNEL_VERSION }}
-            NEW_CHANNEL_VERSION=$((CURRENT_CHANNEL_VERSION+1))
+
+            if [[ ${{ inputs.CHANNEL }}  == "latest" ]]; then
```

I'd suggest we restructure this section to make it easier to maintain. 

```suggestion
function GetNextChannelNumber() {
  case $1 in

  * )
    CURRENT_CHANNEL_VERSION=${{ env.CURRENT_CHANNEL_VERSION }}
    NEW_CHANNEL_VERSION=$((CURRENT_CHANNEL_VERSION+1))
    ELEMENTOR_CHANNEL_PACKAGE_VERSION=${{ inputs.CLEAN_PACKAGE_VERSION }}-${{ inputs.CHANNEL }}${NEW_CHANNEL_VERSION}
    ;;&

  "latest" )
     NEW_CHANNEL_VERSION=-$(date +%s)
     ELEMENTOR_CHANNEL_PACKAGE_VERSION=${{ inputs.CLEAN_PACKAGE_VERSION }}-${{ inputs.CHANNEL }}${NEW_CHANNEL_VERSION}
    ;;

  "ga" )
    ELEMENTOR_CHANNEL_PACKAGE_VERSION=${{ inputs.CLEAN_PACKAGE_VERSION }}
    ;;
  esac

}

GetNextChannelNumber ${{ inputs.CHANNEL }}
```

---

Implemented